javascript/ql/src/semmle/javascript/security/dataflow/Xss.qll

```diff
@@ -0,0 +1,225 @@
+/**
```

Comment:
It is a bit confusing that `Xss.ql` and `Xss.qll` does not correspond directly to each other, but I guess renaming `Xss.ql` to `DomBasedXss.ql` is out of the question for compatibility reasons.

Reply:
We may be able to rename it soon, but I'm not sure whether it's safe to do just yet.